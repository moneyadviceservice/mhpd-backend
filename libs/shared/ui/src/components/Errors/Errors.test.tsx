import React from 'react';
import { render, screen } from '@testing-library/react';
import { Errors } from '.';

describe('Errors component', () => {
  it('renders correctly', () => {
    render(<Errors errors={['errors']}>This contains errors</Errors>);
    const container = screen.getByTestId('errors');
    expect(container).toMatchSnapshot();
  });

  it('renders correctly when there are no errors', () => {
    render(<Errors errors={[]}>This contains errors</Errors>);
    const container = screen.getByTestId('errors');
    expect(container).toMatchSnapshot();
  });
});
