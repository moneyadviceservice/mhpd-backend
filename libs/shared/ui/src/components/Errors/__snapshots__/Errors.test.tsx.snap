// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Errors component renders correctly 1`] = `
<div
  class="border-0 border-l-4 pl-5 border-red-700 border-solid"
  data-testid="errors"
>
  This contains errors
</div>
`;

exports[`Errors component renders correctly when there are no errors 1`] = `
<div
  class=""
  data-testid="errors"
>
  This contains errors
</div>
`;
